src/components/StoriesList/StoriesList.module.scss

```diff
@@ -1,82 +1,130 @@
 @import "../../styles/variables";
 
 .stories {
-  background: $grey;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding-bottom: 50px;
+  padding-bottom: 30px;
+  margin-top: 50px;
+  background: lighten($grey, 10%);
 }
 
 .stories_list {
   display: flex;
+  flex-flow: row wrap;
   justify-content: center;
   max-width: 1000px;
   margin: 0 auto;
 
   a {
-    text-decoration: none;
     display: flex;
     flex-direction: column;
     justify-content: space-between;
     align-items: center;
+    text-decoration: none;
     background: $blue;
-    border-radius: 36px;
-    width: 400px;
+    border-radius: 30px;
+    max-width: 400px;
     height: 330px;
     margin: 20px;
     padding: 20px;
-    transition: 0.3s;
+    box-shadow: 6px 6px $dark-blue;
```

Comment:
Might be worth it to add box shadows at every pixel between 1 and what you have it to ie:

`box-shadow: 6px 6px $dark-blue, 5px 5px $dark-blue, 4px 4px $dark-blue ..... and so on;`

I admit it's an extremely subtle improvement, but it is an improvement.

Reply:
TIL, I love this!